en/documentation/index.md

```diff
@@ -144,9 +149,9 @@ Here is a list of popular tools used by Rubyists:
 
 ### Further Reading
 
-[Ruby-Doc.org][34] maintains a comprehensive list of English
-documentation sources. There are also plenty of [books about Ruby][35].
-If you have questions about Ruby the [mailing
+[Ruby-Doc.org][34] and [Omniref.com][40] both maintain a comprehensive 
```

Comment:
It seems to serve exactly the same purpose as the RubyDoc.org link that
comes right before it -- unless you're referring to the small (and
decidedly non-comprehensive) set of links that ruby-doc has to other
websites on the homepage.

I don't really care if you remove the link there, but it seems like a
double-standard.

On Sun, Feb 22, 2015 at 1:25 PM, Marcus Stollsteimer <
notifications@github.com> wrote:

> In en/documentation/index.md
> <https://github.com/ruby/www.ruby-lang.org/pull/974#discussion_r25136775>:
>
> > @@ -144,9 +149,9 @@ Here is a list of popular tools used by Rubyists:
> >
> >  ### Further Reading
> >
> > -[Ruby-Doc.org][34] maintains a comprehensive list of English
> > -documentation sources. There are also plenty of [books about Ruby][35].
> > -If you have questions about Ruby the [mailing
> > +[Ruby-Doc.org][34] and [Omniref.com][40] both maintain a comprehensive
>
> That's *not* what is meant in the "further reading" paragraph; rather
> tutorials, blog posts, ...
>
> I removed the reference to Omniref from that paragraph.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ruby/www.ruby-lang.org/pull/974/files#r25136775>.
>



-- 
Tim Robertson
Founder, https://www.omniref.com
A code-centric universal reference manual, by and for the programming
community.

Check us out on Twitter: https://twitter.com/omniref
Talk about Omniref on our community subreddit:
http://www.reddit.com/r/omniref